### PR TITLE
Update a handful of NCIT mappings

### PR DESCRIPTION
**Pull request by matentzn**
These are just a few reviewed NCIT mappings I had from an early training with my new contractor.

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -420458,7 +420460,6 @@ xref: ICDO:9834/3 {source="NCIT:C4752"}
 xref: MedDRA:10042985 {source="Orphanet:86871", source="Orphanet:86871/e"}
 xref: MESH:D015461 {source="Orphanet:86871", source="MONDO:equivalentTo", source="Orphanet:86871/e"}
 xref: NCIT:C4752 {source="EFO:1000560", source="MONDO:equivalentTo"}
-xref: NCIT:C70649 {source="MONDO:equivalentObsolete"}
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:420461]: `NCIT:C70649` is not obsolete.

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -419651,7 +419654,6 @@ synonym: "Light chain amyloidosis" RELATED [GARD:0005797]
 synonym: "Light-chain amyloidosis" EXACT [Orphanet:85443]
 synonym: "primary AL amyloidosis" RELATED [GARD:0005797]
 synonym: "primary amyloidosis" EXACT [Orphanet:85443]
-synonym: "primary amyloidosis (formerly)" RELATED [GARD:0005797]
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:419654]: `(formerly)` is an irrelevant part of the label

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -418998,7 +419002,6 @@ xref: ICD9:215.9 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: ICDO:9571/0 {source="NCIT:C4973"}
 xref: MESH:D018317 {source="DOID:4697"}
 xref: NCIT:C4973 {source="DOID:4697", source="MONDO:equivalentTo"}
-xref: NCIT:C6912 {source="DOID:4697"}
```
Comment (matentzn) [inline src/ontology/mondo-edit.obo:419001]: xref moved to soft tissue perineurioma

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -331247,6 +331249,7 @@ subset: orphanet_rare {source="Orphanet:100002"}
 subset: rare
 synonym: "soft tissue perineurioma" EXACT [Orphanet:100002]
 xref: GARD:19730 {source="Orphanet:100002"}
+xref: NCIT:C6912 {source="https://orcid.org/0009-0001-6494-4831"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:331252]: Source `MONDO:equivalentTo` is missing

File: src/ontology/mondo-edit.obo [hunk 5]
```diff
@@ -399398,6 +399401,7 @@ xref: ICD10CM:C25.1 {source="Orphanet:424039/nd", source="Orphanet:424039"}
 xref: ICD10CM:C25.2 {source="Orphanet:424039/nd", source="Orphanet:424039"}
 xref: ICD10CM:C25.7 {source="Orphanet:424039/nd", source="Orphanet:424039"}
 xref: ICD10CM:C25.8 {source="Orphanet:424039/nd", source="Orphanet:424039"}
+xref: NCIT:C173813 {source="https://orcid.org/0009-0001-6494-4831"}
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:399404]: Source MONDO:equivalentTo is missing

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:420461]: I am assuming that we will get `NCIT:C70649` back into Mondo during the alignment process.

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@matentzn 
There are a few "MONDO:equivalentTo" source missing. 
Since this is part of training, I am sending this back to you/your new contractor. (it is helpful to see where the errors/missing things are). 
It is not much to update though, so let me know if you want me to add these.

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (matentzn) [reply src/ontology/mondo-edit.obo:331252]: Can you add these - I overlooked them (I added these by hand from the spreadsheet we worked on). Sorry; Thanks :D

**Comment by matentzn** [comment]
cc @Silvia113-SM

File: src/ontology/mondo-edit.obo (hunk 4 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.obo:331252]: I will add these, no problem! :-)

**Review by sabrinatoro — APPROVED** [review]

**Pull request merged**